submissions/m-ruslan/js-dom/app.js

```diff
@@ -0,0 +1,142 @@
+document.addEventListener("DOMContentLoaded", () => {
+  console.log("DOM is loaded");
+  loadMenu();
+  DEFAULT_PAGE();
+
+  document
+    .querySelector(".nav-container")
+    .addEventListener("click", (event) => {
+      let aEl = event.target.closest("a");
+      if (!aEl) return;
+      let section = aEl.id;
+      loadPageData(section);
+    });
+});
+
+const DEFAULT_PAGE = () => loadPageData("info");
```

Comment:
No, I meant to put `"info"` in a constant:
```javascript
const DEFAULT_PAGE = "info";

loadPageData(DEFAULT_PAGE)
```
This way a) risk of typos is reduced b) if you decide the other page to be default it will be easier to make this change

Reply:
Thank you Anastasiya)
I've changed constant, could you please check?